src/script/lua_api/l_env.cpp

```diff
@@ -1376,6 +1376,29 @@ int ModApiEnv::l_get_translated_string(lua_State * L)
 	return 1;
 }
 
+// change_abm(abm_name, parameters)
+int ModApiEnv::l_change_abm(lua_State *L)
+{
+	GET_ENV_PTR;
+
+	std::string abm_name = readParam<std::string>(L, 1);
+
+	ActiveBlockModifier *abm = env->getActiveBlockModifierNoEx(abm_name);
```

Comment:
wouldn't it be better to have something like `env->replaceActiveBlockModifier` that replaces it a whole?
this would also make it unnecessary to have `change` in the parent class

Reply:
I don't think so. In that case, we will still have to have some method like `getActiveBlockModifierNoEx` to get the original values of fields, that the user does not want to change.

Reply:
you can simply read those from `core.registered_abms[id]`

Reply:
The `change` method was removed.